### ISSUE TITLE
Attached files show up twice in emails (two identical copies)

### ISSUE THREAD

**Issue by cool-RR** [tomekwojcik/envelopes#25]

**Comment by cool-RR** [comment]
_Crickets_

**Comment by cool-RR** [comment]
I know it's only September, but solving this ticket would be a great early Christmas present.

**Comment by Lvl4Sword** [comment]
Using the following:

```
import datetime
import envelopes

def mail_send(files_to_send):

    archive_path = '/home/user/archive/'
    files_to_send = ['1.jpg', '2.jpg', '3.jpg']

    print(files_to_send)

    right_now = datetime.datetime.now()

    envelope = Envelope(
        from_addr = ('edited', 'FROM_NAME'),
        to_addr = ('edited', 'TO_NAME'),
        subject = 'Log Report: {0}'.format(right_now.strftime("%Y/%m/%d")),
        text_body = ""
    )

    for each in files_to_send:
        envelope.add_attachment(archive_path + each)

    envelope.send('smtp.googlemail.com', login='edited',
                  password='edited', tls=True)
```

i get 1.jpg sent twice, so i'm having this same issue.

EDIT: this is just an issue with gmail's interface. you can also verify this by clicking "show original" in gmail, and noticing that it shows no duplicate attachment.

**Comment by Lvl4Sword** [comment]
this is still an issue with envelopes though, and i've sent images using programs/accounts outside of envelopes and they work perfectly. turns out these other images i've sent use "mixed" rather than "alternative" for the multipart. i also found the fix from #27, and believe the fix should be pushed and both of these issues set as solved :+1: 

tl;dr: change msg = MIMEMultipart('alternative') to msg = MIMEMultipart('mixed')
on [line 276 in to_mime_message in envelopes.py](https://github.com/tomekwojcik/envelopes/blob/master/envelopes/envelope.py#L276)
